### DISCUSSION TITLE
Incorrectly Approved as Faculty. I am a Student!

### DISCUSSION THREAD

**Discussion by Sephonix** [community/community#111270] [GitHub Education]
### Hi everyone,

I applied for the GitHub Student Education Pack and was approved on Feb 29, 2024 as a faculty member? **I am not a faulty member,** I am a **student.** Any ideas on how to get this fixed? I just submitted a support ticket but the stories explaining how long it takes to hear back are concerning... Should I resubmit my information?

**Comment by juanpflores** [comment]
Hey there! I've updated your status. It will take a couple of hours to propagate but you should be all set now.
